### PR TITLE
Completed CSS Brochure Form

### PR DESCRIPTION
**Pull request by SharjeelSIqbal**
image

### CODE REVIEW

**Comment by SharjeelSIqbal** [comment]
Better image
image

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  font-weight: bold;
+  border-radius: 3rem;
+  background-color: orangered;
+  color: white;
+  height: 2rem;
+  width: 5rem;
+
+  /*
+  margin-left: 278px;
+*/
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: Please remove commented code prior to submitting for review

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One minor note for you:

**Review by codingcodymiller — APPROVED** [review]
Nice work on this!!! 🎉 🎉 🎉

**Pull request merged**